lib/rails-cloud-tasks.rb

```diff
@@ -49,7 +49,7 @@ def queue_adapter
 
   def self.queue_adapter
     @queue_adapter ||= Adapter.new
-  rescue StandardError => e
+  rescue StandardError, Errno::EHOSTDOWN => e
```

Comment:
Isn't `Errno::EHOSTDOWN` a subclass of `StandardError` already?

Reply:
The exception is not captured

Reply:
Is not. `Errno::EHOSTDOWN ` is an Exception raised by SO

https://www.honeybadger.io/blog/understanding-rubys-strange-errno-exceptions/

Reply:
Then I don't know why rubocop didn't like it..